stellar/deploy-contract.js

```diff
@@ -107,7 +108,7 @@ async function deploy(options, config, chain, contractName) {
             deployer: wallet.publicKey(),
             wasmHash: serializeValue(wasmHash),
         };
-        printInfo(contractName + ' Wasm Hash', serializeValue(wasmHash));
+        printInfo(contractName, JSON.stringify(chain.contracts[contractName], null, 2));
         return;
     }
```

Comment:
remove the if block, this should be done for any deployment, and not just interchain token

Reply:
We should include an `if ... return` block for the interchain_token contract because we're only uploading the wasm (installing) rather than deploying the contract and initializing it with constructor args. So the if block ie necessary here.